docs/modules/spam_checker_callbacks.md

```diff
@@ -58,7 +63,8 @@ async def user_may_invite(inviter: str, invitee: str, room_id: str) -> bool
 
 Called when processing an invitation. The module must return a `bool` indicating whether
```

Comment:
Interestingly this is called for both client-server API and federation, while `user_may_send_3pid_invite` is not (I suspect it doesn't make sense to, not sure it is worth calling out).